source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTableRow.mm

```diff
@@ -86,7 +87,11 @@ - (instancetype)init:(ACOBaseCardElement *)acoElem
                 if (idx < cells.size()) {
                     auto cell = cells.at(idx);
                     ACRTableCellDefinition *cellDefinition = [[ACRTableCellDefinition alloc] init];
-                    cellDefinition.style = (ACRContainerStyle)row->GetStyle();
+                    auto style = row->GetStyle();
+                    if (style == ContainerStyle::None) {
+                        style = cell->GetStyle();
+                    }
+                    cellDefinition.style = (ACRContainerStyle)style;
                     cellDefinition.horizontalAlignment = (ACRHorizontalAlignment)row->GetHorizontalCellContentAlignment();
                     cellDefinition.verticalAlignment = (ACRVerticalAlignment)row->GetVerticalCellContentAlignment();
```

Comment:
The row vertical alignment shouldn't override the cell vertical alignment. It should be the other way around.

Reply:
I agree with you. I had to choose between the two options when we didn't have optionals, and I went with the most straightforward way.